rust/impl/publisher/wpublisher_entry.rs

```diff
@@ -22,11 +22,11 @@ fn main() -> Result< (), wtools::error::BasicError >
 {
   let args = env::args().skip( 1 ).collect::< Vec< String > >();
 
-  let ca = wca::commands_aggregator()
```

Comment:
Please, do not join two chains. It is better to split the instantiating and performing of some action.